RacingCar/Calculator.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Calculator.swift
+//  RacingCar
+//
+//  Created by nylah.j on 2022/04/29.
+//
+
+import Foundation
+
+struct Calculator {
+    func calculate(operators: [Operation], operands: [Int]) throws -> Int {
+        var result = operands[0]
+        
+        for i in 0..<operators.count {
+            result = try operators[i].operate(result, operands[i + 1])
```

Comment:
```suggestion
        for index in 0..<operators.count {
            result = try operators[index].operate(result, operands[index + 1])
```
간결한 코드도 좋지만, Swift는 명료함에 더 초점을 두기에 의견 드려봅니다. 👀